src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -7673,6 +7678,10 @@ final synchronized void removeOpenStatement(ISQLServerStatement st) {
     boolean isAEv2() {
         return (aeVersion >= TDS.COLUMNENCRYPTION_VERSION2);
     }
+    
+    boolean isRetrySupported() {
```

Comment:
NIT: Similar to above. This seems too generic for what it actually applies to.

Reply:
I've changed this to doesServerSupportEnclaveRetry()